include/argon2.h

```diff
@@ -178,6 +178,8 @@ typedef struct Argon2_Context {
     uint8_t *out;    /* output array */
     uint32_t outlen; /* digest length */
 
+    uint32_t version; /*version number*/
```

Comment:
This `version` member would seem to make more sense below, along with all the other Argon2-related hashing settings, instead of mixed with the inputs/outputs. This has no effect on anything, I just found the positioning of this member variable odd.